ci/unit/test_main_node.sh

```diff
@@ -15,6 +15,16 @@ sudo mount -t tmpfs -o size=16G tmpfs /mnt/daos
 sudo mkdir -p "$DAOS_BASE"
 sudo mount -t nfs "$HOSTNAME":"$HOSTPWD" "$DAOS_BASE"
 sudo cp "$DAOS_BASE/install/bin/daos_admin" /usr/bin/daos_admin
+set +x
+if [ -n "$BULLSEYE" ]; then
```

Comment:
Boolean:
```suggestion
if $BULLSEYE; then
```

Reply:
Does not work.
`$ echo $foo

$ if $foo; then echo "true"; else echo "false"; fi
true

$ foo=
$ if $foo; then echo "true"; else echo "false"; fi
true`